src/api.ts

```diff
@@ -9,6 +11,10 @@ export async function fetchProductsPreorderState(handle: string) {
   const resp = await fetch(url.toString());
 
   if (resp.ok) {
+    trackEvent("integration.product_availability_call", {
```

Comment:
I guess there will also be some change in the analytics pipeline to turn these into views or something?

Seems a bit odd for the events to be so specific to the implementation to me. Can this code not just emit the right event type to start with?

Reply:
Yes, but the change is really simple. I'd rather be able to distinguish between the types of events in case this API calls are not a great proxy than treat them as exactly the same.

Reply:
How does this let you know if they are a good proxy or not?